src/__tests__/shape.test.ts

```diff
@@ -16,7 +16,7 @@ describe('shape', () => {
       levels: [0.5],
     });
     expect(timeout).toBeFalsy();
-    expect(contours[0]).toHaveLength(25);
+    expect(contours[0].lines).toHaveLength(25);
```

Comment:
This is a breaking change to the API. It's ok but I'm commenting so we don't forget to put it in the final commit message.

Reply:
Okay thank you